include/sparrow/layout/run_end_encoded_layout/run_end_encoded_iterator.hpp

```diff
@@ -26,16 +26,16 @@ namespace sparrow
 
     // this iteratas over the **actual** values of the run encoded array
     // Ie nullabes values, not values !!!
-    template <bool CONST>
+    template <bool SPARROW_CONST>
```

Comment:
Nitpicking: Can we use IS_CONST instead of SPARROW_CONST. This latter sounds like a sparrow macro definition for `const`

Reply:
Yeah that's actually an argument for switching to lower case `is_const`

Reply:
done